src/nvim/keymap.h

```diff
@@ -240,8 +240,8 @@ enum key_extra {
   , KE_DROP = 95              // DnD data is available
   // , KE_CURSORHOLD = 96     // CursorHold event
   , KE_NOP = 97               // no-op: does nothing
-  , KE_FOCUSGAINED = 98       // focus gained
-  , KE_FOCUSLOST = 99         // focus lost
+  // , KE_FOCUSGAINED = 98    // focus gained
+  // , KE_FOCUSLOST = 99      // focus lost
```

Comment:
These are used in the original vim patch for `gotchars()` but Neovim removed `K_FOCUS` pseudo-keys.